starter-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -193,7 +193,11 @@
             <plugin>
                 <groupId>fish.payara.maven.plugins</groupId>
                 <artifactId>payara-micro-maven-plugin</artifactId>
+#if (${javaVersion} == '8')
```

Comment:
I think it should be 1.4.0
I tested with 2.0 and it also breaks.

Reply:
I have tested Maven Plugin 2.0 with JDK 8 and JDK 11, it looks fine.